Northstar.Client/mod/resource/ui/menus/modlist.menu

```diff
@@ -407,11 +407,11 @@ resource/ui/menus/modlist.menu
 			
 			labelText	"#CLEAR_FILTERS"
 			wide		100
-			xpos		-15
-			ypos		-55
+			xpos		0
+			ypos		0
```

Comment:
I don't see any difference regarding vertical centering here?

Reply:
centering change is only for the Join button in server_browser.menu

Reply:
Where was vertical centring mentioned? Based on the PR description this PR should shift the text a bit to the right to centre it properly

Reply:
idk i didint mention vertical centering

Reply:
`ypos` was modified so I assumed vertical position was changed as well

Reply:
horizontal position wasnt changed

Reply:
It was replaced by the textAlignment I think

Reply:
not replaced

Reply:
change in modlist.menu doesnt have anything to do with centering the join button